airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_30_22_001__Store_last_sync_state.java

```diff
@@ -48,32 +47,13 @@ public class V0_30_22_001__Store_last_sync_state extends BaseJavaMigration {
   static final Field<OffsetDateTime> COLUMN_CREATED_AT = DSL.field("created_at", SQLDataType.TIMESTAMPWITHTIMEZONE);
   static final Field<OffsetDateTime> COLUMN_UPDATED_AT = DSL.field("updated_at", SQLDataType.TIMESTAMPWITHTIMEZONE);
 
-  private final String databaseUser;
-  private final String databasePassword;
-  private final String databaseUrl;
-
-  public V0_30_22_001__Store_last_sync_state() {
-    // EnvConfigs left in place for migration purposes as FlyWay prevents injection, but isolated to
-    // local scope.
-    final EnvConfigs configs = new EnvConfigs();
-    this.databaseUser = configs.getDatabaseUser();
-    this.databasePassword = configs.getDatabasePassword();
-    this.databaseUrl = configs.getDatabaseUrl();
-  }
-
-  @VisibleForTesting
-  V0_30_22_001__Store_last_sync_state(final String databaseUser, final String databasePassword, final String databaseUrl) {
-    this.databaseUser = databaseUser;
-    this.databasePassword = databasePassword;
-    this.databaseUrl = databaseUrl;
-  }
-
   @Override
   public void migrate(final Context context) throws Exception {
     LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
     final DSLContext ctx = DSL.using(context.getConnection());
 
-    final Optional<Database> jobsDatabase = getJobsDatabase(databaseUser, databasePassword, databaseUrl);
+    final Optional<Database> jobsDatabase = getJobsDatabase(context.getConfiguration().getUser(),
```

Comment:
This is a weird dependency here between jobs and configs, I am actually wondering if this is still relevant?

Reply:
@gosusnp Agreed. There are a lot of smells in this particular script that relate to ordering of migration.